lib/api/dsl/match/matchRange.js

```diff
@@ -11,7 +32,11 @@
  * @param {boolean} not - used by notrange operator
  */
 function MatchRange (storage, testTables, document, not = false) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this 'MatchRange' function to match the regular expression ^[_a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS100)